device-types/Synology/RS3614xs+.yaml

```diff
@@ -0,0 +1,50 @@
+---
+manufacturer: Synology
+model: RS3614xs+
+slug: synology-rs3614xs-plus
+part_number: RS3614xs+
+u_height: 2
+is_full_depth: true
+weight: 15.5
+weight_unit: kg
+airflow: front-to-rear
+comments: '[Synology RS3614xs+ Datasheet](https://global.download.synology.com/download/Document/Hardware/DataSheet/RackStation/14-year/RS3614xs+/enu/Synology_RS3614xs_Plus_Data_Sheet_enu.pdf)'
+front_image: true
+rear_image: true
+module-bays:
+  - name: PSU-1
+    label: '1'
+    position: PSU-1
+  - name: PSU-2
+    label: '2'
+    position: PSU-2
+  - name: PCIe Slot 1
+    position: '1'
+  - name: PCIe Slot 2
+    position: '2'
+rear-ports:
```

Comment:
Rear ports are designed for patch panels not devices, standard USB ports on servers are not something that we are currently mapping in this repository